src/absa/data/parser.py

```
@@ -0,0 +1,34 @@
+"""Parse SemEval-2014 Restaurants aspect-term annotations."""
+
+import xml.etree.ElementTree as etree
+from pathlib import Path
+
+from .schema import AspectExample
+
+
+def parse_aspect_examples(path: Path, source_split: str) -> list[AspectExample]:
+    """Return one canonical record per annotated aspect term in *path*."""
+    root = etree.parse(path).getroot()
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)